README.md

```diff
@@ -619,6 +619,26 @@ Running the `telegraf` agent will also clean up the previous metrics, so you are
 > [!Warning]
 > The metrics must be imported after each scenario run and cleanup, since they are associated only to the latest scenario run.
 
+### Import Holochain Metrics
+
+To import Holochain metrics into InfluxDB, holochain must be run with metrics enabled to output to file, and then use `telegraf`, a tool that collects and sends metrics to InfluxDB.
+
+To run Holochain with metrics, the `HOLOCHAIN_INFLUX_FILE` environment variable must be set. For example:
+```bash
+export HOLOCHAIN_INFLUX_FILE=telegraf/metrics/holochain.influx
```

Comment:
```suggestion
export HOLOCHAIN_INFLUX_FILE=$(pwd)/telegraf/metrics/holochain.influx
```

Probably best to provide an absolute path to Holochain, otherwise it will write relative to where the Holochain process is running